src/main/java/Duke/Command/DateOfCommand.java

```diff
@@ -0,0 +1,40 @@
+package Duke.Command;
+
+import Duke.DukeException.DukeIncompleteException;
+import Duke.DukeException.DukeSyntaxErrorException;
+import Duke.Main.TaskList;
+import Duke.Task.Deadline;
+import Duke.Task.Event;
+import Duke.Task.Task;
+
+public class DateOfCommand extends Command {
+
+    private String description;
+    private TaskList taskList;
+    public DateOfCommand(String description, TaskList taskList) {
```

Comment:
Having a bank line separation helps in enhancing readability

Reply:
I noticed the same issue in several other places too

Reply:
Having a bank line separation between the variable and the constructor helps in enhancing readability. I noticed the same issue in several other places too.